rest-api/pom.xml

```diff
@@ -33,7 +33,7 @@
     </parent>
 
     <artifactId>pipeline-rest-api</artifactId>
-    <version>2.2-SNAPSHOT</version>
+    <version>2.2-optimized-SNAPSHOT</version>
```

Comment:
I have an action item to review applicability of jitpack.io for Jenkins plugins.